docs/core/deploying/trimming/trimming-options.md

```diff
@@ -56,7 +100,11 @@ The framework libraries have this attribute. In .NET 6+, you can also opt in to
 
 This is equivalent to setting MSBuild metadata `<IsTrimmable>true</IsTrimmable>` for the assembly in `ManagedAssemblyToLink` (see below).
```

Comment:
Does the "see below" still apply? That info doesn't seem to be in the .NET 7 pivot.

Would it be better to just have a table that showed the different values for different versions?

Reply:
I'll just delete it. We're trying to de-emphasize ManagedAssemblyToLink.